concordia/settings.py

```diff
@@ -262,3 +262,4 @@
     }
 ]
 
+USE_POSTGRES = True if 'postgresql_psycopg2' in Config.Get("database")["adapter"] else False
```

Comment:
This ternary expression is redundant and could simply be:

 `USE_POSTGRES = 'postgresql_psycopg2' in Config.Get("database")["adapter"]`

… or if you want to emphasize the logic using parentheses:

 `USE_POSTGRES = ('postgresql_psycopg2' in Config.Get("database")["adapter"])`

Also since what the config system refers to as an adapter is what Django refers to as an engine, it would be better to use the non-deprecated `django.db.backends.postgresql` value and perhaps make this check more robust:

`USE_POSTGRES = 'postgres' in Config.Get("database")["adapter"]`

Reply:
The DB engine setting has been changed upstream, so we should go with 

`USE_POSTGRES = 'postgres' in Config.Get("database")["adapter"]`

Reply:
Should we keep the “adapter” term or use “engine” to match Django?